### PR TITLE
Edits to Apollo AST article

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

File: docs/source/advanced/apollo-ast.mdx [hunk 1]
```diff
@@ -136,50 +127,54 @@ val schemaIssues = schemaGQLDocument.validateAsSchema()
 println(schemaIssues.map { it.severity.name + ": " + it.message })
 ```
 
-This will print `[WARNING: Unknown directive 'FooBar']`.
+When excuted, this snippet prints `[WARNING: Unknown directive 'private']`.
 
-The `validateAsExecutable` method ensures the operations are consistent with a schema, that is why it takes an instance
-of `Schema` as a parameter. This `Schema` can be obtained by calling the `GQLDocument.toSchema()` extension:
```
Comment (StephenBarlow) [inline docs/source/advanced/apollo-ast.mdx:142]: I'm not sure in Kotlin-land when to refer to something as a method vs. an extension. Is this library adding custom methods to a class that's defined elsewhere?

Comment (BoD) [reply docs/source/advanced/apollo-ast.mdx:142]: 👋  Yes you got it, extensions are when a method (... which are also called functions in Kotlin) is defined not in the class itself, but outside of it. They are used to augment an API you're not the author of (external lib), or sometimes just to organize your code in different modules / files, which is the case here.

To be accurate I used the term extension here, but I think we could have just said method and that would have worked too 😅

Comment (martinbonnin) [reply docs/source/advanced/apollo-ast.mdx:142]: I guess the proper term is "extension function": https://kotlinlang.org/docs/extensions.html#extension-functions

**Review by martinbonnin — APPROVED** [review]

File: docs/source/advanced/apollo-ast.mdx [hunk 2]
```diff
@@ -136,50 +127,54 @@ val schemaIssues = schemaGQLDocument.validateAsSchema()
...
+[WARNING: Use of deprecated field 'height', ERROR: Can't query 'frends' on type 'Character']
+```
 
-This method takes as a parameter a lambda that provides a `GQLNode` and returns the transformed node (or `null` to
-remove the node). It will then visit all nodes in the AST, apply the lambda to each one, and replace the node with the
-result.
+## Outputting SDL
 
-For instance, here's how to remove any field named `foobar` from the selections:
+You can output a `GQLDocument` to standard GraphQL SDL with the `toUtf8` extensions:
```
Comment (martinbonnin) [inline docs/source/advanced/apollo-ast.mdx:150]: Nit, I'd remove "SDL" up here as technically operations are not SDL (as they're not schemas obvisouly). Actually, I'm not exactly sure if SDL is something inside the spec.

```suggestion
You can output a `GQLDocument` to standard GraphQL with the `toUtf8` extensions:
```

Comment (BoD) [inline docs/source/advanced/apollo-ast.mdx:150]:
```suggestion
You can output a `GQLDocument` to standard GraphQL syntax with the `toUtf8` extensions:
```

(as it's not just for SDL)

**Review by martinbonnin — APPROVED** [review]

File: docs/source/advanced/apollo-ast.mdx (hunk 2 above)
Comment (martinbonnin) [reply docs/source/advanced/apollo-ast.mdx:150]: I'll resolve my comment above 😄

Comment (BoD) [reply docs/source/advanced/apollo-ast.mdx:150]: Haha woops we got the same reaction almost at the same minute! 😅 🤝

**Review by BoD — APPROVED** [review]
👍 🙏

**Comment by StephenBarlow** [comment]
Thanks for the review everyone, and for making the followup edit @BoD! I was out the last couple days, ready to jump back into more of these 😄

**Pull request merged**